patternfly/patternfly-elements
# fix reference error in pfe-navigation when slot is missing

### What has changed and why

pfe-navigation contained a reference to a variable that may not exist.  The code was already doing some defensive checks of that kind, but one edge case was missed.  This PR adds the edge case.

    Uncaught (in promise) TypeError: Cannot read property 'length' of undefined

See commit 27aa87d3 to view that change.

I'm calling out the exact commit because this PR also runs prettier to format pfe-navigation which generated a lot of excess diffs.  Somehow this element hasn't been formatted by the automatic formatter already.

### Testing instructions

    npm run build pfe-navigation

Open `elements/pfe-navigation/demo/index.html`, and comment out the `<pfe-navigation-item>` that has `slot="search"`.

    npm start

View the pfe-navigation demo page and confirm that the JS error is no longer in the console.

### Ready-for-merge Checklist
